script/openqa-load-templates

```diff
@@ -101,56 +101,57 @@ my $client = OpenQA::Client->new(apikey => $options{'apikey'}, apisecret => $opt
 my @tables = (qw(Machines TestSuites Products JobTemplates JobGroups));
 
 sub print_error ($res) {
+    my $err;
     if (my $err = $res->error) {
```

Comment:
I guess you wanted?
```suggestion
    my $err;
    if ($err = $res->error) {
```

Reply:
Actually I had something different in mind by turning around the evaluation like `die … unless $err` first but that did not quite work out so I reverted but apparently had a left-over. Updated.